pkg/test/framework/suite.go

```diff
@@ -264,6 +266,20 @@ func (s *suiteImpl) RequireMultiPrimary() Suite {
 	return s
 }
 
+func (s *suiteImpl) IgnoreConfigClusterTopology() Suite {
```

Comment:
```suggestion
func (s *suiteImpl) RequireInClusterConfig() Suite {
```

maybe? I was gonna say it should begin with `Require` but the wording is tricky. I think `Skip` is better than `Ignore` though.